### PR TITLE
Added Logger info to changelog

### PR DESCRIPTION
**Pull request by burner**
Some shameless self promoting on my part. Still IMO worth mentioning.

### CODE REVIEW

**Comment by Hackerpilot** [comment]
Yes, this should be in the changelog.

File: changelog.dd [hunk 1]
```diff
@@ -183,6 +184,17 @@ $(LI $(LNAME2 aa-keyvalue, $(D byKeyValue) was added.)
 )
 )
 
+$(LI $(LNAME2 logger, An experiemental logging system was added.)
```
Comment (JakobOvrum) [inline changelog.dd]: s/experiemental/experimental/

Comment (burner) [inline changelog.dd]: thank you

File: changelog.dd [hunk 2]
```diff
@@ -183,6 +184,17 @@ $(LI $(LNAME2 aa-keyvalue, $(D byKeyValue) was added.)
...
+
+------
+import std.experimental.logger;
+
+void main()
+{
+    log("message logging in D");
+}
+------
+
```
Comment (MartinNowak) [inline changelog.dd]: You're missing a closing paren for the LI, like everyone else who submitted a changelog entry ;).

Comment (burner) [inline changelog.dd]: I will fix that

**Comment by MartinNowak** [comment]
Please rebase

**Comment by burner** [comment]
rebased

**Comment by burner** [comment]
thanks

**Pull request merged**